docs/guide/docs/channels/web.md

```diff
@@ -280,3 +280,124 @@ const payload = {
 ```
 
 [security sdk]: https://botpress.io/reference/modules/_botpress_sdk_.security.html#getmessagesignature
+
+# Customizing Web Chat Style
```

Comment:
I think this is an addition to the already existing "Custom look and feel" section. I'd merge both of them.